### ISSUE TITLE
Wrong order of dataset scale and shift in model training

### ISSUE THREAD

**Issue by jhbastek** [BaratiLab/Diffusion-based-Fluid-Super-resolution#11]
In `train_ddpm/runners/diffusion_tub.py` the loss is computed as

`loss = loss_registry[config.model.type](model, x, t, e, b, x_offset.item(), x_scale.item())`.

This calls `def conditional_noise_estimation_loss(model,
                          x0: torch.Tensor,
                          t: torch.LongTensor,
                          e: torch.Tensor,
                          b: torch.Tensor,
                          x_scale,
                          x_offset,
                          keepdim=False, p=0.1)` in `train_ddpm/functions/losses.py` and thus swaps `x_scale` with `x_offset`. It would be much appreciated if the authors could double check if they used a correct version for the results in the paper or these results suffer from this mismatch. Thank you in advance.
